IntegrationTests/GoldMaster/Braintree.podspec.json.goldmaster

```diff
@@ -606,7 +606,7 @@ acknowledged_target(
   ],
   value = "//Vendor/Braintree/pod_support_buildable:acknowledgement_fragment"
   )
-objc_bundle_library(
+apple_resource_bundle(
```

Comment:
So this is migrating to the new `rules_apple` rule? Awesome 👍 

Eventually, we may need to add a call to ```
load("//apple:resources.bzl", "apple_resource_bundle")
```
e.g. in [BuildFile.swift])(https://github.com/pinterest/PodToBUILD/blob/7defad37813a8e03261e98198d13d48c4d534533/Sources/PodToBUILD/BuildFile.swift#L110)

For now, I think since they've added the new rule names into Bazel as well, it's good not to have it https://github.com/bazelbuild/bazel/blob/c67dceca77a93646961fc4bb863826e344fda5e0/src/main/java/com/google/devtools/build/lib/rules/objc/ObjcRuleClasses.java#L459